db/src/commonMain/kotlin/com/islamversity/db/model/EntityUUID.kt

```diff
@@ -0,0 +1,18 @@
+package com.islamversity.db.model
+
+import com.squareup.sqldelight.ColumnAdapter
+
+data class EntityUUID(
+    val id  : String
+){
+    val isValid : Boolean
```

Comment:
Useless

Reply:
needs to be used by creator of the this class